### DISCUSSION TITLE
Custom preset not showing up.

### DISCUSSION THREAD

**Discussion by Menisco** [nroduit/Weasis#339] [Q&A]
Hello, I'm trying to implement a custom preset, because one of our hospitals uploads funky images and it's annoying to change the window/level for every single DICOM.

I tried adding the following line to the presets archive but it didn't show up as an option after opening the DICOM:
`<preset name="HRSP" modality="MG" window="165" level="165"/>`

Changing the modality to CT or RM made it show up and function as intended on DICOM's of those modalities.
I thought it might be the DICOM archive modality, but it seems it is correctly assigned as MG.

So, am I doing something wrong or is the problem really in the files?
Is there a way to implement a preset available for all modalities?

Thanks in advance!

![Screenshot_2](https://user-images.githubusercontent.com/96140007/193830265-4e99ac20-bde9-4fb2-9ff1-c8f716bc41a5.png)
![Screenshot_3](https://user-images.githubusercontent.com/96140007/193830274-7452ca25-81fd-4476-b6ee-5a75ba75bb08.png)

**Comment by nroduit** [comment]
I don't see any issue to add a MG preset:
![image](https://user-images.githubusercontent.com/993975/194769714-aa8642e6-f9e3-46c3-ad1d-65e846a4b531.png)
